src/xdocs/config_annotation.xml

```diff
@@ -228,6 +228,111 @@ public void test(&#64;MyAnnotation String s) { // OK
       </subsection>
     </section>
 
+    <section name="AnnotationOnSameLine">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.2</p>
+        <p>
+          Check annotation located on same line with its target.
```

Comment:
Please make this description the same the check's javadoc.

Reply:
Updated to javadoc text